<launch>
  <node pkg="tfmini_sensor" type="tfmini_sensor_node" name="tfmini_sensor_node">
    <param name="serial_port" type="string" value="/dev/ttyUSB0" />
    <param name="baud_rate" type="int" value="115200" />
  </node>
  
  <node pkg="tf" type="static_transform_publisher" name="broadcastertfmini" args="0 0 0 0 0 0 base_link tfmini 100" />
</launch>
